src/redux/businessPages/businessPages.sagas.js

```diff
@@ -0,0 +1,51 @@
+import { takeEvery, call, put } from 'redux-saga/effects';
+import { push } from 'connected-react-router';
+
+import { setBusinessPage, setLoading } from './businessPages.actions';
+import { GET_BUSINESS_PAGE_BY_CODE } from './businessPages.types';
+import { setError } from '../error/error.actions';
+import getItems from '../../utils/client';
+
+export function* handleBusinessPageLoad({ payload }) {
+  console.log(payload);
+  try {
+    yield put(setLoading(true));
+    const businessPage = yield call(
+      getItems,
+      `query{
+            getBusinessTextByCode(code: "${payload}") {
```

Comment:
Don't use string concatenations in graphql, provide the value via `$variables`, it is a more convenient and safe way